Dockerfile

```diff
@@ -21,8 +21,9 @@ EXPOSE 4000
 CMD ["poetry", "run", "flask","run","--host=0.0.0.0"]
 
 FROM base as production
-CMD ["poetry", "run","gunicorn", "todo_app.app:app", "-b", "0.0.0.0:5000"]
+# CMD ["poetry", "run","gunicorn", "todo_app.app:app", "-b", "0.0.0.0:80"]
 
+CMD poetry run gunicorn todo_app.app:app -b 0.0.0.0:$PORT
```

Comment:
Note that this change will break your existing prod container instructions in your Readme since they don't pass through a "PORT" environment variable. One way to solve that is to set a default value in the Dockerfile which can be overridden but means if none is set it will still default to 5000:
```
ENV PORT=5000
```